images/build

```diff
@@ -68,6 +68,9 @@ for IMAGE in "${!IMAGE_USERS[@]}"; do
     build_image ./image-user "$IMAGE" --build-arg "USER=$USER"
 done
 
+# Build the user with pre-defined group image
+build_image ./image-predefined-group image-predefined-group
+
```

Comment:
I added this image for the new validation case. Could someone tell me how to publish this image??

Reply:
We build images for each commit via Google Cloud Build: https://github.com/kubernetes-sigs/cri-tools/blob/master/images/build

Adding the image there should work.

Reply:
Thanks, I found typo 🙇 I will give it another try.